torch/_dynamo/output_graph.py

```diff
@@ -1373,6 +1373,9 @@ def cleanup(self) -> None:
         self.input_name_to_proxy.clear()
         self.side_effects.clear()
         self.register_finalizer_fns.clear()
+        # None is invalid for these, and clearing them messes up the user scope.
```

Comment:
Just for my understanding, are you saying we have to reset these by redefining them to be empty instead of calling clear on them? Why does clearing them mess up user scope?